### PR TITLE
[PLAY-727] Lightbox currentPhotoIndex Prop

### PR DESCRIPTION
**Pull request by nidaqg**
**What does this PR do?** A clear and concise description with your runway ticket url.
No Runway ticket, this story adds the currentPhoto prop that will allow users to change slide using external controls. 

**Screenshots:** Screenshots to visualize your addition/change

**How to test?** Steps to confirm the desired behavior:
4. See addition/change

#### Checklist:
- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

**Review by jasperfurniss — DISMISSED** [review]

File: playbook/app/pb_kits/playbook/pb_lightbox/docs/_lightbox_current_photo.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                  onClick={() => setCurrentPhoto(active < photos.length - 1 ? active + 1 : photos.length - 1)}
+              >
+                Next
+              </Button>
+            </div>
+          </div>
+          <Lightbox
+              currentPhoto={currentPhoto}
+              icon="times"
+              initialPhoto={selectedPhoto}
```
Comment (drborges) [inline playbook/app/pb_kits/playbook/pb_lightbox/docs/_lightbox_current_photo.jsx:77]: @nidaqg is there any limitation that would preventing us from simplifying the API a little by removing `initialPhoto` and having simply `currentPhoto`? Or `currentPhotoIndex`?

Comment (nidaqg) [reply playbook/app/pb_kits/playbook/pb_lightbox/docs/_lightbox_current_photo.jsx]: @drborges I've removed `initialPhoto` prop and changed the `currentPhoto` prop name to `currentPhotoIndex`. I'm going to cut another alpha with these latest changes to make sure that removing the `initialPhoto` prop doesn't break anything in pulse-ui where this kit is being used

**Review by jasperfurniss — APPROVED** [review]

**Review by drborges — APPROVED** [review]
Thanks, this should allow us to implement the use case that requires external slide controls.

File: playbook/app/pb_kits/playbook/pb_lightbox/Carousel/index.tsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import Slides from './Slides'
 import Thumbnails from './Thumbnails'
 
 type CarouselType = {
+  initialPhoto?: string,
```
Comment (jasperfurniss) [inline playbook/app/pb_kits/playbook/pb_lightbox/Carousel/index.tsx]: Awesome! Thanks for fixing this. Marking here for visibility.

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**